src/utils/threads/thread_pool.cpp

```diff
@@ -63,20 +63,30 @@ void thread_pool::launch_pinned_threads(
 
   auto error = pthread_getaffinity_np(pthread_self(),
                                       sizeof(cpu_set_t), &cpuset);
+
```

Comment:
use strerror(error) instead of error 
for reporting errors with both thread_getaffinity_np() and thread_setaffinity_np()